### PR TITLE
Correct rspec-rails usage

### PR DESCRIPTION
**Pull request by JonRowe**
The usage instructions for rspec (via `rspec-rails`) are slightly out of date, this PR is an improvement on those.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,16 +27,19 @@ Or install it yourself as:
...
+To work around the issue right now, use a beta version of rspec-rails or
+manually you'll have to include the modules in your `rails_helper`.
 
 ```ruby
 RSpec.configure do |config|
-  config.include Rails::Controller::Testing::TestProcess
-  config.include Rails::Controller::Testing::TemplateAssertions
-  config.include Rails::Controller::Testing::Integration
+  [:controller, :view, :request].each do |type|
+    config.include ::Rails::Controller::Testing::TestProcess, :type => type
```
Comment (tgxworld) [inline README.md:39]: > Use Ruby >= 1.9 syntax for hashes. Prefer { a: :b } over { :a => :b }.

Small nitpick. `type: type`? :grin:

Comment (JonRowe) [inline README.md:39]: If thats what you prefer I can change this, but it's a copy and paste from the rspec-rails beta (we support at least parsing by 1.8.7 hence the hash syntax).

Comment (tgxworld) [inline README.md:39]: Ah ok. Then that is fine :)

**Comment by tgxworld** [comment]
Thank you @JonRowe :) :+1:

**Pull request merged**